### ISSUE TITLE
Data Retention Policy for Epicenter

### ISSUE THREAD

**Issue by jamesmrollins** [lzim/teampsd#1579]
Hi @lzim 

- It came to my attention that Forio wants to implement new policies that do not persist Epicenter stored data indefinitely (they warrant 12 months).  
- We have a large license with them, but Will Glass-Husain indicates that data persistence past one year, would/could be managed with an enterprise agreement.
- I have been clear with them, please don't do anything with our data until we determine our way forward.  They have agreed, and indicate in any circumstance, "There is *no danger*.  Your data is safe. All we are doing is assessing what your needs are."
- I have opened this issue, so we don't loose sight of it.  I don't think we have to anything immediately; however, I do believe we could come up with a different way to persist the data. 
- One option may be to automate the export process, so during the save process, data is moved on one click from the team's Epicenter instance, to a more persistent team location inside the VA domain.
- An automatic data transfer scheme might also be constructive to the development of a more automated Team Time report.  But likely not without significant rewrite of the r-code driving that report.

**Comment by lzim** [comment]
@jamesmrollins 

Thanks for bringing this to our attention.
I agree that it may warrant consideration with automation of exporting team_time (tt) reports.

**Comment by lzim** [comment]
@jamesmrollins @staceypark 

Let's consider this along with tt_report automation during our January roadmap planning.

I do think that 12 months is probably too short for many teams. However, I am thinking that perhaps 2 years would be sufficient for teams being able pick back up again from where they last left off. That is, as long as they have an archive of their last tt report.

**Comment by lzim** [comment]
@jamesmrollins @staceypark @lijenn 

Can we please makes sure we understand the policy and document it for 
- Learners
- Facilitators
- Team PSD?  

Then, close this card.

**Comment by jamesmrollins** [comment]
@lzim I will need to get back to Forio to see where they landed on this. I'm not sure any action is required by us, but will check it out and update.

**Comment by lzim** [comment]
@jamesmrollins  Just a re-ping on this since we have a wave of sites wrapping up and want to instruct about data retention.

FYI: @staceypark & @lijenn

**Comment by jamesmrollins** [comment]
@lzim wilco.

**Comment by jamesmrollins** [comment]
Pinged Will at Forio to ask about retention policy changes.

**Comment by jamesmrollins** [comment]
Hi @lzim 
- I don't think this issue is relevant at this point and recommend we close it.
- The more I looked into it, we really don't put a huge data demand on Forio, because the size of our Team Data and Experiment files are really small. 
- If we held the Data UI in the Sim UI or had a scheme, were we mirrored and transformed large amounts of enterprise data on the Epicenter platform then I think we would have something to worry about.
- Certainly, if it were a problem, I am certain that Will would be calling me to resolve.
